test/cases/testdata/graphql/test-graphql-is-schema-valid.yaml

```diff
@@ -0,0 +1,115 @@
+# This suite of tests is adapted from the underlying GraphQL parser library's
+# own test suite, as it provides a fairly comprehensive set of good/degenerate
+# test cases, which we want to make sure to react correctly to.
+# See: https://github.com/vektah/gqlparser/blob/master/validator/validator_test.yml
+cases:
```

Comment:
Only happy path tests here, AFAICS.

Reply:
I think the reason for that is because the original test suite I borrowed from for the `graphql.is_valid` builtin was just happy-path tests too. I can fix that though!

Reply:
Added a stack of tests, adapted from the `gqlparser` test suite. Marking as resolved. :+1:

Reply:
Committed, too? I can't find it 😅 but it's early, probably just not seeing it.

Reply:
Found it. Ignore me.